busted/outputHandlers/json.lua

```diff
@@ -1,25 +1,39 @@
-local json = require 'dkjson'
+local json = require("dkjson")
 local io_write = io.write
 local io_flush = io.flush
 
 return function(options)
-  local busted = require 'busted'
-  local handler = require 'busted.outputHandlers.base'()
+  local busted = require("busted")
+  local handler = require("busted.outputHandlers.base")()
 
   handler.suiteEnd = function()
     local error_info = {
       pendings = handler.pendings,
       successes = handler.successes,
       failures = handler.failures,
       errors = handler.errors,
-      duration = handler.getDuration()
+      duration = handler.getDuration(),
```

Comment:
Same as above, lets back out the style-only changes so we only have the functional ones related to this PR and address the style issues is bulk elsewhere.